cloud/amazon/ec2_elb_lb.py

```diff
@@ -142,6 +143,21 @@
     required: false
     default: 60
     version_added: "2.1"
+  tags:
+    description: the tags dict to apply
+    type: dict
+    required: false
+    sample: { "Department": "Information Retrieval" }
+    version_added: "2.1"
+  purge_tags:
```

Comment:
ah yes. that makes perfect sense.  i will go ahead with the append_tags solution. thanks @bcoca